docs/modules/servers/pages/postgres/objectives.adoc

```diff
@@ -0,0 +1,22 @@
+= Distributed James Server &mdash; Objectives and motivation
+:navtitle: Objectives and motivation
+
+From the outstanding advantages of a distributed mail system, such as scalability and enhancement,
+this project aims to implement a backend database version using Postgres,
+
+Primary Objectives:
+
+* Provide more options: The current James Distributed server uses Cassandra as the backend database.
+  This project aims to provide an alternative to Cassandra, using Postgres as the backend database.
+  The choice of version depends on the operator's decision.
```

Comment:
Not feeling right with this item list. Distributed server is not the only option to run James (memory, jpa, smtp only, ...).

More like trying to offer a higly scalable and reactive James mail server like distributed but for small and medium deployments (while distributed is obviously more for big ones?)